src/packet/decode.ts

```diff
@@ -0,0 +1,96 @@
+import RLP = require("rlp");
```

Comment:
import rlp from "rlp" (or * as rlp) ?

Reply:
I don't think rlp is browser safe

https://www.npmjs.com/package/rlp-browser

Reply:
I'm not sure we can even use Discv5 in the browser, because you would need to proxy through a rendezvous server.

Reply:
RLP is browser compatible. Not a fan of using this fork (that isn't as well maintained as the original), we can use a web bundler to provide the Buffer polyfill.